src/functional_pipeline/functional_pipeline.m

```diff
@@ -62,20 +65,63 @@
 
 
 % PARSE INPUT
-[subjectDir, varargin{:}] = convertStringsToChars(subjectDir, varargin{:});
+[subjectSelection, varargin{:}] = convertStringsToChars(subjectSelection, varargin{:});
 
-assert(ischar(subjectDir), ...
+assert(ischar(subjectSelection), ...
     'CATO:functional_pipeline:subjectDirNotText', ...
     'subjectDir must be a row vector of characters or string scalar.');
-assert(isdir(subjectDir), ...
+assert(isdir(subjectSelection) | isfile(subjectSelection), ...
```

Comment:
The function `isfolder` is recommended instead of `isdir`:
[https://www.mathworks.com/help/matlab/ref/isdir.html](https://www.mathworks.com/help/matlab/ref/isdir.html)